### ISSUE TITLE
Videos on Android are black using UaaL, works fine in pure Unity build.

### ISSUE THREAD

**Issue by adavies25** [Unity-Technologies/uaal-example#49]
Hi, I'm using UaaL with Unity 2020.3.14f1 and AR Foundation 4.1.7 and the video I'm trying to play is just a VideoClip using the Unity VideoPlayer component and it shows a black screen where the video should be. the video plays fine in an Android build built directly from Unity. In the logcat, it's spewing this message:

W/Unity: AndroidVideoMedia::OpenExtractor could not translate sharedassets0.resource to local file. Make sure file exists, is on disk (not in memory) and not compressed.
    AndroidVideoMedia: Error opening extractor: -10004

I found it's similar to this issue playing video in compressed asset bundles, they have to be uncompressed.
[https://issuetracker.unity3d.com/issues/android-loading-videos-from-assetbundles-fails-unless-its-uncompressed-and-loaded-from-file](https://issuetracker.unity3d.com/issues/android-loading-videos-from-assetbundles-fails-unless-its-uncompressed-and-loaded-from-file)

So I'm wondering if it needs to be extracted and/or uncompressed first from sharedassets0? Is there a workaround like putting the video in StreamingAssets and playing it from a URL?
Thanks,
Alan

**Comment by julien-tamade** [comment]
Hey @adavies25 did you ever solve this? Also running into video issues on android in my project.

**Comment by adavies25** [comment]
No. I don't know what was wrong. I haven't tried it lately either. Are you
getting the same error message? Did you see the unity issue I linked?

**Comment by timbotimbo** [comment]
Another workaround is to stop gradle from compressing the video file.
Add noCompress to app/build.gradle:
```
android {
 
    lintOptions {...}
    defaultConfig {...}
	
    aaptOptions {
        noCompress = ['.resource']
    }
}
```
